services/appmotion_gateway/template.api_env

```diff
@@ -0,0 +1,80 @@
+
+#
+# env to compose with repo.config and resolve into .env (i.e. configs.payments_gateway_env)
+#
+
+APP_NAME="${APPMOTION_GATEWAY_APP_NAME}"
+APP_ENV="${APPMOTION_GATEWAY_APP_ENV}"
+APP_KEY="${APPMOTION_GATEWAY_APP_KEY}"
+APP_DEBUG="${APPMOTION_GATEWAY_APP_DEBUG}"
+APP_URL="${APPMOTION_GATEWAY_APP_URL}"
+
+LOG_CHANNEL="${APPMOTION_GATEWAY_LOG_CHANNEL}"
+LOG_DEPRECATIONS_CHANNEL="${APPMOTION_GATEWAY_LOG_DEPRECATIONS_CHANNEL}"
+LOG_LEVEL="${APPMOTION_GATEWAY_LOG_LEVEL}"
+
+DB_CONNECTION="${APPMOTION_GATEWAY_DB_CONNECTION}"
+DB_HOST="${APPMOTION_GATEWAY_DB_HOST}"
+DB_PORT="${APPMOTION_GATEWAY_DB_PORT}"
+DB_DATABASE="${APPMOTION_GATEWAY_DB_DATABASE}"
+DB_USERNAME="${APPMOTION_GATEWAY_DB_USERNAME}"
+DB_PASSWORD="${APPMOTION_GATEWAY_DB_PASSWORD}"
+
+BROADCAST_DRIVER="${APPMOTION_GATEWAY_BROADCAST_DRIVER}"
+CACHE_DRIVER="${APPMOTION_GATEWAY_CACHE_DRIVER}"
+FILESYSTEM_DISK="${APPMOTION_GATEWAY_FILESYSTEM_DISK}"
+QUEUE_CONNECTION="${APPMOTION_GATEWAY_QUEUE_CONNECTION}"
+SESSION_DRIVER="${APPMOTION_GATEWAY_SESSION_DRIVER}"
+SESSION_LIFETIME="${APPMOTION_GATEWAY_SESSION_LIFETIME}"
+
+MEMCACHED_HOST="${APPMOTION_GATEWAY_MEMCACHED_HOST}"
+
+REDIS_HOST="${APPMOTION_GATEWAY_REDIS_HOST}"
+REDIS_PASSWORD="${APPMOTION_GATEWAY_REDIS_PASSWORD}"
+REDIS_PORT="${APPMOTION_GATEWAY_REDIS_PORT}"
+
+MAIL_MAILER="${APPMOTION_GATEWAY_MAIL_MAILER}"
+MAIL_HOST="${APPMOTION_GATEWAY_MAIL_HOST}"
+MAIL_PORT="${APPMOTION_GATEWAY_MAIL_PORT}"
+MAIL_USERNAME="${APPMOTION_GATEWAY_MAIL_USERNAME}"
+MAIL_PASSWORD="${APPMOTION_GATEWAY_MAIL_PASSWORD}"
+MAIL_ENCRYPTION="${APPMOTION_GATEWAY_MAIL_ENCRYPTION}"
+MAIL_FROM_ADDRESS="${APPMOTION_GATEWAY_MAIL_FROM_ADDRESS}"
+MAIL_FROM_NAME="${APPMOTION_GATEWAY_MAIL_FROM_NAME}"
+
+AWS_ACCESS_KEY_ID="${APPMOTION_GATEWAY_AWS_ACCESS_KEY_ID}"
+AWS_SECRET_ACCESS_KEY="${APPMOTION_GATEWAY_AWS_SECRET_ACCESS_KEY}"
+AWS_DEFAULT_REGION="${APPMOTION_GATEWAY_AWS_DEFAULT_REGION}"
+AWS_BUCKET="${APPMOTION_GATEWAY_AWS_BUCKET}"
+AWS_USE_PATH_STYLE_ENDPOINT="${APPMOTION_GATEWAY_AWS_USE_PATH_STYLE_ENDPOINT}"
+
+PUSHER_APP_ID="${APPMOTION_GATEWAY_PUSHER_APP_ID}"
+PUSHER_APP_KEY="${APPMOTION_GATEWAY_PUSHER_APP_KEY}"
+PUSHER_APP_SECRET="${APPMOTION_GATEWAY_PUSHER_APP_SECRET}"
+PUSHER_HOST="${APPMOTION_GATEWAY_PUSHER_HOST}"
+PUSHER_PORT="${APPMOTION_GATEWAY_PUSHER_PORT}"
+PUSHER_SCHEME="${APPMOTION_GATEWAY_PUSHER_SCHEME}"
+PUSHER_APP_CLUSTER="${APPMOTION_GATEWAY_PUSHER_APP_CLUSTER}"
+
+VITE_APP_NAME="${APPMOTION_GATEWAY_VITE_APP_NAME}"
+VITE_PUSHER_APP_KEY="${APPMOTION_GATEWAY_VITE_PUSHER_APP_KEY}"
+VITE_PUSHER_HOST="${APPMOTION_GATEWAY_VITE_PUSHER_HOST}"
+VITE_PUSHER_PORT="${APPMOTION_GATEWAY_VITE_PUSHER_PORT}"
+VITE_PUSHER_SCHEME="${APPMOTION_GATEWAY_VITE_PUSHER_SCHEME}"
+VITE_PUSHER_APP_CLUSTER="${APPMOTION_GATEWAY_VITE_PUSHER_APP_CLUSTER}"
+
+STRIPE_KEY="${APPMOTION_GATEWAY_STRIPE_KEY}"
+STRIPE_SECRET="${APPMOTION_GATEWAY_STRIPE_SECRET}"
+
+SESSION_SECURE_COOKIE="${APPMOTION_GATEWAY_SESSION_SECURE_COOKIE}"
+
+PAYMENT_WEHOOK="${APPMOTION_GATEWAY_PAYMENT_WEHOOK}"
```

Comment:
please review these variables. 
1. `WEHOOK` (no `S`) is propbably wrong
2. ALL envars of this service are prefixed by `APPMOTION_GATEWAY_` (i.e. NO `payment`)
3. My original PR kept some of these variables set to a fixed value in the template. (e.g. `MAIL_MAILER` )  Why did you add variables for **all** of them? IMO this is unecessary since even some of the environs are not used and only adds more complexity

Reply:
Fixed